static/src/javascripts/projects/common/modules/commercial/acquisitions-ophan.js

```diff
@@ -5,11 +5,13 @@ const submitComponentEvent = (componentEvent: OphanComponentEvent) => {
     ophan.record({ componentEvent });
 };
 
-export const submitInsertEvent = (
+export const submitEvent = (
```

Comment:
maybe we could call this `submitComponentEvent` and rename the private function?